lab_1_classify_by_unigrams/main.py

```diff
@@ -65,6 +96,22 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     In case of corrupt input arguments, None is returned
     """
 
+    if (not isinstance(predicted, list)
+            or not isinstance(actual, list)):
+        return None
+
+    n = len(actual)
```

Comment:
rename this variable.

Reply:
the name of the variable should make it clear what it is responsible for

Reply:
it is n-value from the MSE formula from Wiki

Reply:
done